addons/danger/functions/fnc_assaultBuilding.sqf

```diff
@@ -66,7 +66,7 @@ if (RND(0.95) || {_unit distance (_buildingPos select 0) < 3.2}) then {
     _buildingPos deleteAt 0;
 
     // update variable
-    _building setVariable ["LAMBS_CQB_cleared_" + str (side _unit), _buildingPos];
+    _building setVariable [QGVAR(CQB_cleared_) + str (side _unit), _buildingPos];
```

Comment:
What about setting this public so that it is shared over Multiple Zeus for example?

Reply:
I would like to push that for post-2.0

Reply:
yeah thats fine 👍